solarkraft/src/verify.ts

```diff
@@ -147,16 +154,25 @@ export function verify(args: any) {
                     console.log(
                         `${path.basename(monitorPath)}: ${errorStringOrOK}`
                     )
+                    apalacheResult
+                        .map(() => (verificationStatus = 'ok'))
```

Comment:
I wouldn't expect an assignment inside the functional part. Isn't `apalacheResult` redundant after this? Why not return the result of this map/mapLeft?